src/Ledger/Ratify.lagda

```diff
@@ -377,20 +376,14 @@ accepted' Γ (record { cc = cc , _ ; pparams = pparams , _ }) gs =
 
     acceptedBy : GovRole → Set
     acceptedBy role =
-      let t = maybe id R.0ℚ $ threshold pparams (proj₂ <$> cc) action role in
+      let t = maybe id 0ℚ $ threshold pparams (proj₂ <$> cc) action role in
       case totalStake role cc' redStakeDistr votes' of λ where
-        0 → t ≡ R.0ℚ -- if there's no stake, accept only if threshold is zero
-        x@(suc _) → Z.+ acceptedStake role cc' redStakeDistr votes' R./ x R.≥ t
+        0 → t ≡ 0ℚ -- if there's no stake, accept only if threshold is zero
+        x@(suc _) → ℤ.+ acceptedStake role cc' redStakeDistr votes' ℚ./ x ℚ.≥ t
```

Comment:
`ℚ.≥` could also be handled by the new "order" typeclasses, namely defining `≥ = flip ≤` under `HasPreorder`.

This of course suggests there are many other derived definitions that are currently missing; feel free to get some inspiration from my prelude: [Prelude.Ord.Core](https://omelkonian.github.io/formal-prelude/Prelude.Ord.Core.html), [Prelude.Ord.Dec](https://omelkonian.github.io/formal-prelude/Prelude.Ord.Dec.html), [Prelude.Ord.MinMax](https://omelkonian.github.io/formal-prelude/Prelude.Ord.MinMax.html).

It would be too much to put these in the current PR though, so contribute these in a subsequent PR if needed.

Reply:
I actually tried, briefly, to make a type class instance for rationals, but there are some things missing in the standard library that we use to prove `≤⇔<∨≈` in the Nat and Int instances.  I'll make an issue about it for future work.